mne/viz/epochs.py

```diff
@@ -1056,7 +1056,7 @@ def _plot_onkey(event, params):
     elif event.key == '-':
         params['scale_factor'] /= 1.1
         _plot_traces(params)
-    elif event.key == '+':
+    elif event.key == '+' or event.key == '=':
```

Comment:
Zen of python says: `There should be one-- and preferably only one --obvious way to do it.` :)